USNRewardPayOut.sol

```diff
@@ -0,0 +1,32 @@
+import "./SampleOffer.sol";
+
+
+contract USNRewardPayOut {
+     
+     SampleOffer public usnContract;
+     
+     function USNRewardPayOut(SampleOffer _usnContract) {
+          usnContract = _usnContract;
+     }
+
+     // interface for USN
+    function payOneTimeReward() returns(bool) {
+        if (msg.value < usnContract.getDeploymentReward())
```

Comment:
If the deployment reward is strictly related to the USN, why not having it in this contract?